docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -284,6 +284,13 @@ Congratulations, you've deployed a basic Dockerized Alluxio cluster! Read on to
 
 ## Advanced Setup
 
+### Launch Alluxio with the development image
+Starting from v2.6.2, we have another docker image, alluxio-dev, available in Dockerhub for development usage. Unlike the default alluxio/alluxio image that 
```

Comment:
Extra line between title and paragraph

Reply:
Starting from v2.6.2, a new docker image, alluxio-dev, is available  ....
try to reduce we/I in the doc